TPMCmd/Platform/src/NVMem.c

```diff
@@ -63,7 +63,7 @@ static int NvFileOpen(const char* mode)
 {
 #  if defined(NV_FILE_PATH)
 #    define TO_STRING(s)      TO_STRING_IMPL(s)
-#    define TO_STRING_IMPL(s) #s
+#    define TO_STRING_IMPL(s) #    s
```

Comment:
This looks suspicious - the #s should be string-izing the input, and I'm not sure that will happen with the space.

If you ran clang-format.  Would like to know which version?

Reply:
Thanks.  This turns out to be a bug in clang-format.
https://github.com/llvm/llvm-project/issues/56255 
Works on 15.0.1.